### PR TITLE
Make Player::peer_id server-side only and add getters and setters

### PR DESCRIPTION
**Pull request by nerzhul**
Player::peer_id has no sense client side, move it to server, make it private and add setter and getter
Also add some PEER_ID_INEXISTENT instead of harcoded 0

### CODE REVIEW

**Comment by rubenwardy** [comment]
> Make Player::peer_id server-side only, and add getters and setters

**Comment by nerzhul** [comment]
@sfan5 @rubenwardy appart the commit msg are you okay with it ? :)

File: src/content_sao.cpp [hunk 1]
```diff
@@ -1359,7 +1359,7 @@ void PlayerSAO::disconnected()
 void PlayerSAO::unlinkPlayerSessionAndSave()
 {
 	assert(m_player->getPlayerSAO() == this);
-	m_player->peer_id = 0;
+	m_player->setPeerId(0);
```
Comment (SmallJoker) [inline src/content_sao.cpp]: `PEER_ID_INEXISTENT`

File: src/serverenvironment.cpp [hunk 2]
```diff
@@ -431,8 +431,8 @@ void ServerEnvironment::addPlayer(RemotePlayer *player)
 		Exception: there can be multiple players with peer_id=0
 	*/
 	// If peer id is non-zero, it has to be unique.
-	if (player->peer_id != 0)
-		FATAL_ERROR_IF(getPlayer(player->peer_id) != NULL, "Peer id not unique");
+	if (player->getPeerId() != 0)
```
Comment (SmallJoker) [inline src/serverenvironment.cpp]: `PEER_ID_INEXISTENT`

File: src/script/lua_api/l_server.cpp [hunk 3]
```diff
@@ -171,14 +171,17 @@ int ModApiServer::l_get_player_information(lua_State *L)
...
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::AVG_RTT, &avg_rtt))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MIN_JITTER,
+		&min_jitter))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::MAX_JITTER,
+		&max_jitter))
+	ERET(getServer(L)->getClientConInfo(player->getPeerId(), con::AVG_JITTER,
+		&avg_jitter))
+
+	ERET(getServer(L)->getClientInfo(player->getPeerId(),
 										&state, &uptime, &ser_vers, &prot_vers,
```
Comment (SmallJoker) [inline src/script/lua_api/l_server.cpp]: As you're changing/correcting this function, could you please also fix the indents here?

**Pull request merged**